proto/internalapi/central/hello.proto

```diff
@@ -7,30 +7,36 @@ import "storage/cluster.proto";
 package central;
 
 message HelmManagedConfigInit {
-  storage.CompleteClusterConfig cluster_config = 1;
-  string cluster_name                          = 2;
-  string cluster_id                            = 3;
-  bool not_helm_managed                        = 4 [deprecated = true]; // Only taken into account if managed_by is UNKNOWN.
-  storage.ManagerType managed_by               = 5;
+    storage.CompleteClusterConfig cluster_config = 1;
+    string cluster_name = 2;
+    string cluster_id = 3;
+    bool not_helm_managed = 4 [deprecated = true]; // Only taken into account if managed_by is UNKNOWN.
+    storage.ManagerType managed_by = 5;
```

Comment:
Strange that the indents to align the numbers were removed 🤔

Reply:
My auto format does this. I manually aligned the equal signs.